Essentials/src/com/earth2me/essentials/commands/Commandnuke.java

```diff
@@ -26,7 +26,7 @@ protected void run(final Server server, final CommandSource sender, final String
         if (args.length > 0) {
             targets = new ArrayList<>();
             int pos = 0;
-            for (String arg : args) {
+            for (int i = 0; i < args.length; i++) {
```

Comment:
Why?

Reply:
If you are trying to remove the pos environment variable, you should uh, _actually remove it_.